stdlib/public/Darwin/Foundation/NSDictionary.swift

```diff
@@ -13,6 +13,13 @@
 @_exported import Foundation // Clang module
 import _SwiftFoundationOverlayShims
 
+// We don't check for NSCopying here for performance reasons. We would
+// just crash anyway, and NSMutableDictionary will still do that when
+// it tries to call -copyWithZone: and it's not there
+func duckCastToNSCopying(_ x: Any) -> NSCopying {
```

Comment:
```suggestion
private func duckCastToNSCopying(_ x: Any) -> NSCopying {
```

Any reason it needs to be internal?

Reply:
Nope, good catch